dw-snackbar.js

```diff
@@ -268,23 +234,66 @@ export class DwSnackbar extends layoutMixin(LitElement) {
      * by the user. So, the Toast submitted first will be shown on the top in the sequence.
     */
     this.constructor.counter = 0;
+    this.positionVertical = 'bottom';
+
+    window.addEventListener('resize', this._getLayout.bind(this))
+  }
+
+  _getLayout() {
```

Comment:
This method name is misleading. From it's name it looks like it's read action, but it's logic is write action (sets `mobile` property)